### DISCUSSION TITLE
[Feature] Detects file renaming / moving on an external library

### DISCUSSION THREAD

**Discussion by Elerir** [immich-app/immich#13037] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Prevents duplicates / pictures "not found" due to renaming or moving a file in the picture directory

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by etnoy** [comment]
This is documented not to be supported yet, see https://immich.app/docs/features/libraries/

**Reply by Elerir** [reply]
It is indeed documented :

> If an external asset is deleted from disk, Immich will move it to trash on rescan. To restore the asset, you need to restore the original file. After 30 days the file will be removed from trash, and any changes to metadata within Immich will be lost.

So I expected the problem to be solved after 30 days. To make sure, I checked my trash after a full rescan : it is empty
